src/main/java/duke/io/UserInput.java

```diff
@@ -0,0 +1,90 @@
+package duke.io;
+
+/**
+ * This is a UserInput class that store the user's input to Duke.
+ * This class helps Duke to make sense of the user's input.
+ */
+public class UserInput {
+    private String type = "";
+    private String command = "";
+    private String description = "";
+    private String time = "";
```

Comment:
Is there a reason you initialized these variables to empty strings? You could leave them uninitialized maybe?

Reply:
It's just my habit to make sure none of the variables are uninitialised in the later part of my code.